go/kbnm/hostmanifest/whitelist.go

```diff
@@ -0,0 +1,61 @@
+package hostmanifest
+
+import (
+	"encoding/json"
+	"os"
+	"os/user"
+	"path/filepath"
+)
+
+// whitelistPath is used for installing the whitelist as a JSON into a given
+// path on disk.
+type whitelistPath struct {
+	// Root is the path of the NativeMessage whitelists for root user.
```

Comment:
Honestly, I'm not 100% sure. I suspect it's implementation-specific, but I imagine the relevant implementations do what you say. I'll update the comment.